### PR TITLE
[AIRFLOW-3341] FAQ return DAG object example

### PR DESCRIPTION
**Pull request by kik-kik**
### Description
- Added a small code example of a function returning a DAG object to `FAQ` section.

### Tests
- No unit tests required as this is a doc update.

### Commits
- Added an example function returning a DAG object

### Documentation
- Not applicable

### Code Quality

- `flake8` - not applicable

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -147,10 +147,21 @@ simple dictionary.
...
+
+        return DAG(dag_id)
+
+
     for i in range(10):
         dag_id = 'foo_{}'.format(i)
         globals()[dag_id] = DAG(dag_id)
+
         # or better, call a function that returns a DAG object!
+        other_dag_id = 'dag_id_{}'.format('returned')
```
Comment (mik-laj) [inline docs/faq.rst]:
```suggestion
        other_dag_id = 'bar_{}'.format(i)
```
dag id must be unique.

**Review by mik-laj — CHANGES_REQUESTED** [review]
The idea is right. It is worth adding this example, but you have to improve it.

File: docs/faq.rst (hunk 1 above)
Comment (kik-kik) [reply docs/faq.rst]: Apologies for the late response, been away on a long holiday and after had loads on my plate.

**Review by mik-laj — APPROVED** [review]

**Comment by mik-laj** [comment]
@potiuk PTAL

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Can you please rebase to the latest master?

**Comment by kik-kik** [comment]
> Can you please rebase to the latest master?

@potiuk DONE

**Comment by potiuk** [comment]
Thanks @kik-kik

**Pull request merged**